javascript/javascript1/week1/script.js

```diff
@@ -0,0 +1,42 @@
+//A future age calculator
+const yearOfBirth = 1986
+const yearFuture = 2027
+my = yearFuture - yearOfBirth
+console.log("You will be " + my + " in 2027")
+
+//A dog age calculator 
+const dogAge = 1957
+const humanAge = 2017
+const futureAge = 2027
+const resultInDogYear = true
+const resultInHumanYear = false
+sumDogAge = futureAge - dogAge
+sumHumanAge = futureAge - humanAge
+
+if(resultInDogYear){
+    console.log("Your dog will be " + sumDogAge + " dog years old in 2027");
+} else if(resultInHumanYear){
+    console.log("Your dog will be " + sumHumanAge + " human years old in 2027");
+}
+
+//Housey pricey
+const wide = 5
+const deep = 11 
+const high = 18
+const priceRange = 2000000
+gardenSizeInM2 = high * deep
```

Comment:
This is not valid since the variable is not declared, and same for `volumeInMeters` and `housePrice`